docs/tutorials/schools.md

```diff
@@ -87,7 +87,7 @@ which emits
 Nothing too tricky here.  After a quick review of the shapes and types,
 you will notice they are just three relatively simple tables, which is no surprise
 since we obtained the original data from
-[SQLite database files](https://github.com/brimdata/zed/blob/main/testdata/edu/README.md#creation).
+[SQLite database files](https://github.com/brimdata/zed/blob/main/testdata/edu/README.md).
```

Comment:
This one is also subtle/annoying. The tl;dr is that it looks like the limitation https://github.com/raviqqe/muffet/issues/52 that I've bumped into before. I don't love changing the links to make the link checker happy, but this anchor destination is close enough to the top of the page that it barely makes a difference.